haiku-apps/beshare/beshare-2.31.recipe

```diff
@@ -49,19 +53,17 @@ BUILD_PREREQUIRES="
 	makefile_engine
 	"
 
-PATCHES="beshare-$portVersion.patchset"
-
 BUILD()
 {
 	ln -sf ../../sources-2/muscle muscle
```

Comment:
Sure, but not all programs use getopt. Some programs use long option names with a single dash, so not merging single-letter options together is imho a nice way to attract new contributors.
It does not cost much to have clearly separated options, and it reduces the risk of visitors saying to themselves: "ah, I definitely don't understand scripts" when reading all these merged options.
Making the effort of being novice-friendly in our recipes can only increase the audience.

Of course, I'm not telling we should change all scripts. I just think that when we change a line that has such merged options, we could, while at it, also switch them to separate options.

That said, I do admit that if this ever increases the audience, the increase will probably be very small.
I also would like to say that this is just a suggestion, and I personally don't care if we use merged options. I just think that a system which targets average users should try to use, when possible, novice-friendly scripts, even for things (e.g. recipes) that these users are probably not going to read.